-------------------------------------------------------------------------------
Test set: UnitTest.StoreUnitTest
-------------------------------------------------------------------------------
Tests run: 11, Failures: 6, Errors: 0, Skipped: 0, Time elapsed: 0.009 s <<< FAILURE! - in UnitTest.StoreUnitTest
testEditProductPrice_Success  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <150.0> but was: <20.0>
	at UnitTest.StoreUnitTest.testEditProductPrice_Success(StoreUnitTest.java:118)

testGetStore_StoreExists  Time elapsed: 0.001 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <DomainLayer.backend.StorePackage.Store@6aba2b86> but was: <DomainLayer.backend.StorePackage.Store@158da8e>
	at UnitTest.StoreUnitTest.testGetStore_StoreExists(StoreUnitTest.java:42)

testGetInfo  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <No such store!> but was: <>
	at UnitTest.StoreUnitTest.testGetInfo(StoreUnitTest.java:153)

testAddProduct_StoreDoesNotExist  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.lang.Exception to be thrown, but nothing was thrown.
	at UnitTest.StoreUnitTest.testAddProduct_StoreDoesNotExist(StoreUnitTest.java:71)

testEditProductQuantity_Success  Time elapsed: 0.001 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <20> but was: <10>
	at UnitTest.StoreUnitTest.testEditProductQuantity_Success(StoreUnitTest.java:130)

testCloseStore_Success  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <Store Closed Successfully> but was: <Store Closed Successfuly>
	at UnitTest.StoreUnitTest.testCloseStore_Success(StoreUnitTest.java:137)

